File: CSharp/Palindrome.cs
using System;
public class IsPalindrome
{
public static void Main(string[] args)
{
int no,total=0,t,rev;
Console.Write("Please provide the no: ");
no = int.Parse(Console.ReadLine());
t=no;
while(no>0)
{
rev=no%10;
total=(total*10)+rev;
no=no/10;
}
if(t==total)
Console.Write("The given number is Palindrome.");
else
Console.Write("The given number is not Palindrome");
}
}

File: JAVA/Mathematics/palindrome.java

public class palindrome {

    static boolean checkPalindrome (int x){
       boolean flag=false;
        int rev=0;
        int y=x;
        while(x>0){
            rev=rev*10+x%10;
            x=x/10;
        }
        if(y==rev){
            flag=true;
        }
return flag;
    }

    public static void main(String[] args) {
        if(checkPalindrome(1214)){
            System.out.println("Palindrome No");
        }else {
            System.out.println("Not a Palindrome no.");
        }
    }

}
